tests/test_date.py

```diff
@@ -33,6 +33,11 @@
 )
 
 
+def test_loads_validate_dates():
+    with pytest.raises(ValueError):
+        PartialDate.loads('2015-1-1')
+
+
```

Comment:
can you add a test case for the example I gave you?

Reply:
updated

Reply:
that doesn't work. The first call will raise, so the second one will never be executed. You need to split it into two.